templates/blog-page.html

```diff
@@ -15,24 +15,24 @@ <h1>{{ page.title }}</h1>
 </div>
 
 {{ page.content | safe }}
+<div class="blog-authors">
+  <h2>About the authors</h2>
+  {% if page.authors %}
+    {% for author in page.authors %}
+      {% set author_path = "authors/" ~ author ~ ".md" %}
+      {% set author_page = get_page(path=author_path) %}
+      {% include "includes/author_panel.html" %}
+    {% endfor %}
+  {% endif %}
+</div>
 <hr />
 {% include "includes/share-buttons.html" %}
 <div class="blog-nav">
   {% if page.higher and page.higher.title %}
   <a class="prev" href="{{page.higher.permalink}}">&laquo; {{page.higher.title}}</a>
   {% endif %}
   {% if page.lower and page.lower.title %}
-  <a class="next" href="{{page.lower.permalink}}">{{page.lower.title}} &raquo; </a>
+  <a style="float:right;" class="next" href="{{page.lower.permalink}}">{{page.lower.title}} &raquo; </a>
```

Comment:
Inline styles are gauche, hard to maintain, and you can save a few bytes with cached CSS. 

For the same effect, you can revert it to the previous, and add `_valkey.scss`
```css
.next {
  float: right;
}
```